js/app.js

```
@@ -209,8 +234,12 @@ class CampaignManager {
           break;
         case "edit-character":
         case "edit-npc":
-          const characterCard = event.target.closest(".card-character");
-          const characterId = characterCard?.dataset.characterId;
+          let characterCard = event.target.closest(".card-character");
+          let characterId = characterCard?.dataset.characterId;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)